flow/designs/nangate45/ariane136/rtlmp_config.txt

```diff
@@ -0,0 +1,38 @@
+min_aspect_ratio = 0.40
```

Comment:
@ravi-varadarajan I took reference from gf12 for this setup.
But how to generate `rtlmp_config.txt` for technology specific?